tests/core/snap-auto-mount/task.yaml

```diff
@@ -18,9 +18,9 @@ systems:
   - ubuntu-core-16-64
   - ubuntu-core-18-64
   - ubuntu-core-20-64
-  # TODO: auto-import is broken on jammy
-  # see https://bugs.launchpad.net/snapd/+bug/1964596
-  # - ubuntu-core-22-64
+  - ubuntu-core-22-64
+# does not work on core24
```

Comment:
I think it would make sense to add a comment explaining what you put in the PR comment regarding USB/nested.